src/components/enterprise-page/EnterprisePage.jsx

```diff
@@ -3,21 +3,21 @@ import PropTypes from 'prop-types';
 import { useParams } from 'react-router-dom';
 import { identify } from 'react-fullstory';
 import { getAuthenticatedUser } from '@edx/frontend-platform/auth';
-import { AppContext } from '@edx/frontend-platform/react';
+import { AppContext, ErrorPage } from '@edx/frontend-platform/react';
 import { getConfig } from '@edx/frontend-platform/config';
 
 import { LoadingSpinner } from '../loading-spinner';
 import NotFoundPage from '../NotFoundPage';
 
-import { isDefined, isDefinedAndNull, isDefinedAndNotNull } from '../../utils/common';
+import {isDefined, isDefinedAndNull, isDefinedAndNotNull } from '../../utils/common';
```

Comment:
GH Actions shows a linting error here, needs a space after `{`

Reply:
bleh, I thought I fixed all of them